uiowa/uiowa
# GCLID/FBCLID workaround

Resolves https://github.com/uiowa/uiowa/issues/5964

This PR introduces a workaround for Acquia varnish not passing certain click identifiers to a full page load for Drupal Webform's prepopulate functionality to utilize. Instead this attaches JS that is on top of a varnish request that will fire and populate these fields.

# To Test

- In Chrome (not Brave), visit the form with query parameters attached to the URL (e.g. https://sandbox.dev.drupal.uiowa.edu/click-id-form-test?gclid=thisismygclidvalue&fbclid=thisismyfbclidvalue&other_hidden=asdf) and inspect the hidden elements to see that their values are populated.
- If successful, go back and try adjusting configuration for the forms where not all of these conditions match (different element machine names, prepopulate not checked).
